### PR TITLE
fusee-interfacee-tk: init at V1.0.0

### PR DESCRIPTION
**Pull request by Flameslice**
Package fusee-interfacee-tk utility to applications/misc for use with RCM Nintendo Switch
###### Motivation for this change

Add fusee-interfacee-tk package
Add fusee-interfacee-tk to all-applications
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -6680,6 +6682,8 @@ in
 
   Sylk = callPackage ../applications/networking/Sylk {};
 
+  otter-browser = qt5.callPackage ../applications/networking/browsers/otter {};
```
Comment (sikmir) [inline pkgs/top-level/all-packages.nix]: otter-browser?

Comment (Kristian-Brucaj) [reply pkgs/top-level/all-packages.nix]: I have no idea where that came from? I'll remove that.

Comment (Kristian-Brucaj) [reply pkgs/top-level/all-packages.nix]: It has been axed.

**Comment by marsam** [comment]
I'm so sorry for the late response. LGTM, thank you!

**Pull request merged**